### ISSUE TITLE
Error: undefined method `each_pair' for #<HTTParty::Response:0x007fdb8e05fa70> when pulling site via siteleaf CLI

### ISSUE THREAD

**Issue by Pakmandesign** [siteleaf/siteleaf-gem#75]

**Comment by sskylar** [comment]
@Pakmandesign can you provide more context here?

**Comment by brandonarbini** [comment]
```
✔ ~/Code/yellowco.co [master|✔] 
13:52 $ siteleaf push
Reading site...
Uploading _authors/abigail-davidson.markdown...complete.
Uploading _authors/kari-enge.markdown...complete.
Uploading _drafts/gc-interview-draft.markdown...complete.
Uploading _webpack/scss/css/_fotorama.scss...complete.
Uploading blog/category/conscious-living.markdown...complete.
Uploading blog/category/culture.markdown...complete.
Uploading blog/category/entrepreneurs.markdown...complete.
Uploading blog/category/growth.markdown...complete.
Uploading blog/category/health.markdown...complete.
Uploading blog/category/relationships.markdown...complete.
Uploading blog/index.markdown...complete.
Uploading conference/index.markdown...complete.
Uploading index.markdown...complete.
Uploading membership/index.markdown...complete.
Uploading node_modules/@babel/code-frame/lib/index.js...Error: undefined method `each_pair' for #<HTTParty::Response:0x007fabda488e78>
Did you mean?  each_slice
```

Should I be excluding `node_modules`?

**Comment by sskylar** [comment]
@brandonarbini This should be fixed with v2.1.0, but it is generally recommended to exclude `node_modules`.

You can do that by adding it to your `_config.yml` file like this:

```yaml
exclude:
  - Gemfile.lock
  - node_modules
  - vendor/bundle/
  - vendor/cache/
  - vendor/gems/
  - vendor/ruby/
```
